### PR TITLE
Portfolio -Mobile Menu

### PR DESCRIPTION
**Pull request by harshi0102**
The portfolio mobile menu using HTML, CSS and JavaScript is made that finally showcases the dynamic and responsive webpage. 
-Using hamburger menu three menus gets available like portfolio, about and contact 
-There is a cross button also and on clicking on it , the menu gets disappeared.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,15 +41,15 @@
...
               <a
                 href="javascript:void(0);"
                 onclick="menuOptions('#my-recent-work')"
-                >Projects</a
-              >
+                >Portfolio</a>
+              
             </li>
             <li>
               <a href="javascript:void(0);" onclick="menuOptions('#about')"
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Well done :+1: your project all functionalities are working but there are some design issues, please stick to the Figma design check below the comparison between your design and Figma design :point_down: 

**Your Design**
Screenshot from 2023-03-13 13-55-09

**Figma Design**
Screenshot from 2023-03-13 14-06-56

- [ ] If you check your `font style` and Figma design `font style` it does not match so kindly fix this issue. :slightly_smiling_face: 
- [ ] If you check the spacing between the text it does not match to the Figma design so please fix this issue. :+1:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @harshi0102 and @faisalakb, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters are checked
✔️ Nice implementation

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
Please check the PR. I already updated the above comments.

File: index.html [hunk 2]
```diff
@@ -40,15 +41,15 @@
...
           <ul id="mylink">
             <li>
               <a
                 href="javascript:void(0);"
                 onclick="menuOptions('#my-recent-work')"
-                >Projects</a
-              >
+                >Portfolio</a>
+              
             </li>
```
Comment (joel-kalema) [inline index.html]: - [ ] Good job implementing the mobile menu 😄, but as the previous reviewer said If you check your `font style` and Figma design `font style` it does not match so kindly fix this issue. 👍 
> It has to be something like `font-weight: 600;`

- [ ] Also,  If you check the spacing between the text it does not match to the Figma design so please fix this issue. 👍

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |mobilefig| |mobile| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @harshi0102 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @harshi0102 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 👍🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @harshi0102 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

Your design looks good 💯 ✔️
Good PR description ✔️
Follow GitHub flow ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**